app/views/admin/organizations/index.html.erb

```diff
@@ -36,7 +36,7 @@
                 <div class="col-3">
                   <%= render_filterrific_spinner %>
                 </div>
-                <div clas="col-9">
+                <div class="col-12">
```

Comment:
From the previous review by @cielf, this was the only typo fix that caused a regression. But changing the css class fixed the issue.